proto/proxy/destination.proto

```diff
@@ -100,6 +101,13 @@ message TlsIdentity {
     // The name of the pod.
     string pod_name = 3;
   }
+
+  // Verify the certificate based on the Kubernetes pod identity.
+  // The pod_identity string is of the format:
+  // {pod_name}.{pod_type}.{pod_ns}.conduit-managed.{controller_ns}.svc.cluster.local
+  message K8sPodIdentity {
+    string pod_identity = 1;
```

Comment:
Replacing the pod name and namespace is exactly right.

We still need the controller namespace broken out as a separate field because we do a "same controller" check using it.

Reply:
Ah, thanks -- I added a `controller_ns` field in 0b942cf8.